pyproject.toml

```
@@ -62,7 +62,7 @@ test = [
 # release scripts and process.
 dev-release = [
     "towncrier ~= 23.6",
-    "pydoctor ~= 23.9.0",
+    "pydoctor >= 24.11.0",
```

For our dev dependencies, we do a strict pin, trying to reduce the posibility of breaking the CI due to future pydoctor updates.

```suggestion
    "pydoctor ~= 24.11.0",
```